shanoir-ng-datasets/src/main/java/org/shanoir/ng/processing/carmin/controller/CarminDatasetProcessingApiController.java

```diff
@@ -97,13 +105,32 @@ public ResponseEntity<Void> updateCarminDatasetProcessing(
 
     @Override
     public ResponseEntity<List<CarminDatasetProcessing>> findCarminDatasetProcessings() {
-        final List<CarminDatasetProcessing> carminDatasetProcessings = carminDatasetProcessingService.findAll();
+        final List<CarminDatasetProcessing> carminDatasetProcessings = carminDatasetProcessingSecurityService.filterCarminDatasetList(carminDatasetProcessingService.findAll(), RIGHT_STR);
```

Comment:
Dear Alae, thank you for your PR. We normally try to put all SecurityServices into annotations, either PreAuthorize or PostAuthorize. In this case I would propose PostAuthorize. Normally the controller implementation should not need a reference to the SecService. As example:
@PostAuthorize("hasRole('ADMIN') or @datasetSecurityService.**filterDatasetDTOList**(returnObject.getBody(), 'CAN_SEE_ALL')")